feed/feed_test.go

```diff
@@ -399,6 +423,29 @@ func TestPreScoredGenerator(t *testing.T) {
 			},
 			expectedPosts: []string{nsfwArtPost, nsfwLabelledPost},
 		},
+		{
+			name: "all videos",
+			opts: preScoredGeneratorOpts{
+				Alg: "classic",
+				generatorOpts: generatorOpts{
+					Hashtags: []string{},
+					HasMedia: tristate.True,
+				},
+			},
+			expectedPosts: []string{videoPost, nsfwVideoPost},
+		},
+		{
+			name: "tagged videos nsfw",
+			opts: preScoredGeneratorOpts{
+				Alg: "classic",
+				generatorOpts: generatorOpts{
+					Hashtags: []string{"art", "furryart"},
+					IsNSFW:   tristate.True,
+					HasMedia: tristate.True,
```

Comment:
Should this be HasVideo instead of HasMedia ?

Reply:
I think so yeah!